src/core/Akka.Cluster.Tests/StartupWithOneThreadSpec.cs

```diff
@@ -25,12 +27,13 @@ public class StartupWithOneThreadSpec : AkkaSpec
             akka.actor.default-dispatcher.dedicated-thread-pool.thread-count = 1
             akka.actor.provider = ""Akka.Cluster.ClusterActorRefProvider, Akka.Cluster""
             akka.remote.dot-netty.tcp.port = 0
+            akka.cluster.downing-provider-class = ""Akka.Cluster.SBR.SplitBrainResolverProvider, Akka.Cluster""
```

Comment:
Added SBR to the single thread startup spec to see if the deadletter'd `TimerMsg`s would show up here.